R/update_snapshot.R

```diff
@@ -10,8 +10,8 @@
 #' @param tic
 #'   A timestamp when computation began. If not supplied, it will be created at call-time.
 #'   (Used to more accurately convey how long runtime of the update process has been)
-#' @template log_path
-#' @template log_table_id
+#' @param logger
+#'   A [Logger]. If none is given, one is initialized with default arguments.
```

Comment:
Consider writing "A [Logger] instance." to be more concise about the input.

Reply:
Good point. I've done this.